packages/eas-cli/src/commands/diagnostics.ts

```diff
@@ -20,6 +22,7 @@ export default class Diagnostics extends EasCommand {
       nonInteractive: true,
     });
 
+    const spinner = ora().start(`Gathering diagnostic information...`);
     const info = await envinfo.run(
       {
         System: ['OS', 'Shell'],
```

Comment:
Maybe we should `try / catch` it and call `spinner.failed` when there is an error?

Reply:
AFAIK `envinfo` always "succeedes" and only prints error messages instead of throwing on read failures.
* https://github.com/tabrindle/envinfo